tests/models/test_work.py

```diff
@@ -1117,7 +1117,9 @@ def assert_time_match(python, postgres):
             eq_(pool.suppressed, match['suppressed'])
             eq_(pool.data_source_id, match['data_source_id'])
 
+            assert isinstance(match['available'], bool)
             eq_(pool.licenses_available > 0, match['available'])
+            assert isinstance(match['licensed'], bool)
```

Comment:
These assertions don't fail even when the bug is present, but I figured it can't hurt to add them.